### PR TITLE
DCS-2023 pre arrival single match person summary

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: server/routes/bookedtoday/summaryWithRecordController.test.ts [hunk 1]
```diff
@@ -62,66 +62,27 @@ describe('GET /prisoner/:id/summary-with-record', () => {
...
-
-      return request(app)
-        .get('/prisoners/1111-1111-1111-1111/summary-with-record')
-        .expect(200)
-        .expect('Content-Type', 'text/html; charset=utf-8')
-        .expect(res => {
-          const $ = cheerio.load(res.text)
-          expect($('[data-qa=prisoner-image]').attr('src')).toEqual('/assets/images/placeholder-image.png')
+          expect($('[data-qa=prisoner-image]').attr('src')).toEqual('/prisoners/A5534HA/image')
         })
```
Comment (louisenorris) [inline server/routes/bookedtoday/summaryWithRecordController.test.ts:73]: removed test as will always have image for single match

File: server/routes/bookedtoday/summaryWithRecordController.test.ts [hunk 2]
```diff
@@ -62,66 +62,27 @@ describe('GET /prisoner/:id/summary-with-record', () => {
...
-          const $ = cheerio.load(res.text)
-          expect($('span.govuk-caption-l').text().trim()).toStrictEqual('Prison number: A1234AB')
-        })
-    })
-
-    it('should render only PNC Number when only PNC Number is given', () => {
-      expectedArrivalsService.getArrival.mockResolvedValue(withMatchType(createArrival({ prisonNumber: null })))
-
+    it('should render both PNC Number and Prison Number from single potential match', () => {
       return request(app)
```
Comment (louisenorris) [inline server/routes/bookedtoday/summaryWithRecordController.test.ts:79]: removed test as will always display both prison number and PNC number for single match

File: server/views/pages/bookedtoday/summaryWithRecord.njk [hunk 3]
```diff
@@ -18,11 +18,9 @@
         <div class="govuk-body">
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-one-half">
-                    {% if singleMatch.prisonNumber or singleMatch.pncNumber %}
                         <span class="govuk-caption-l">
-                            {% if singleMatch.prisonNumber %}Prison number: {{ singleMatch.prisonNumber }}{% endif %}{% if singleMatch.prisonNumber and singleMatch.pncNumber %} | {% endif %}{% if singleMatch.pncNumber %}PNC: {{ singleMatch.pncNumber }}{% endif %}
+                            Prison number: {{ singleMatch.prisonNumber }} | PNC: {{ singleMatch.pncNumber }}
```
Comment (louisenorris) [inline server/views/pages/bookedtoday/summaryWithRecord.njk:22]: removed conditional as prison number and PNC number always present for single match

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/views/pages/bookedtoday/summaryWithRecord.njk [hunk 4]
```diff
@@ -80,7 +78,7 @@
                 <div class='govuk-grid-column-one-third'>
                     <div class="profile-align-image">
                         <img
-                            src="{{ '/prisoners/' + arrival.prisonNumber + '/image' if arrival.prisonNumber else '/assets/images/placeholder-image.png' }}"
+                            src="{{ '/prisoners/' + singleMatch.prisonNumber + '/image' if singleMatch.prisonNumber else '/assets/images/placeholder-image.png' }}"
```
Comment (andrewrlee) [inline server/views/pages/bookedtoday/summaryWithRecord.njk:81]: no need for conditional?

Comment (louisenorris) [reply server/views/pages/bookedtoday/summaryWithRecord.njk:81]: true

Comment (louisenorris) [reply server/views/pages/bookedtoday/summaryWithRecord.njk:81]: will merge this and fix on the next pr - hope that's ok even though it's for the next ticket

Comment (louisenorris) [reply server/views/pages/bookedtoday/summaryWithRecord.njk]: removed in last commit

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**